SVOM Mission Page (#2637)
nasa-gcn/gcn.nasa.gov: app/routes/missions.svom/route.mdx

---
handle:
  breadcrumb: SVOM
---

import logo from './logo_svom_black.svg'

# Space Variable Objects Monitor (SVOM)

<img
  src={logo}
  align="right"

>   width="250"

Author: ```suggestion
  width="818"
```

>   height="165"

Reviewer: Size does not match image.

>   height="165"

Author: ```suggestion
  height="780"
```

  alt="SVOM logo"
  className="grid-col-6 mobile-lg:grid-col-4 tablet:grid-col-2 desktop:grid-col-3 height-auto"
/>

**Launch Date:** June 22, 2024

**Anticipated Start of Science Operations:** December 2024

**End of Operations:** Mission duration of 3 years

**Data Archive:**

> [HEASARC SVOM Archive](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/svom.html)

Author: @timroland please check this

> The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://irfu.cea.fr/dap/en/Phocea/Vie_des_labos/Ast/ast_technique.php?id_ast=2276). SVOM mission is dedicated to the study of the the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).

Reviewer: The CNES link was broken. Here is the correct link:

```suggestion
The [Space-based multi-band astronomical Variable Objects Monitor (SVOM)](https://www.svom.eu/en/the-svom-mission/) is a French-Chinese mission, result of a collaboration between the two national space agencies, [China National Space Administration (CNSA)](https://www.cnsa.gov.cn/english/index.html) and [Centre national d'études spatiales (CNES)](https://cnes.fr/projets/svom). SVOM mission is dedicated to the study of the the most powerful transient phenomena, with a particular emphasis on gamma-ray bursts (GRBs).
```

The SVOM spacecraft carries four multi-wavelength instruments: ECLAIRs, Gamma Ray burst Monitor (GRM), Microchannel X-ray Telescope (MXT) and Visual Telescope (VT). ECLAIRs and GRM can detect gamma-ray transient sources in real-time with localisation capabilities for ECLAIRs. SVOM can transfer the alerts in near real-time with a typical latency of less than 30s for 2/3 of the alerts and perform autonomous slew for performing x-ray and optical follow-up of the source with the smaller field of view of theMXT and VT instrument

 <div className="overflow-table">

| Instruments | Energy Range                                           | Field of View | Localization                                   |
| ----------- | ------------------------------------------------------ | ------------- | ---------------------------------------------- |

> | ECLAIRs     | 4 - 250 keV                                            | 2 ster        | ~8' radius (statistical, 90% confidence)       |
> | GRM         | 15 - 5000 keV                                          | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
> | MXT         | 0.2 - 10 keV                                           | 1.1° x 1.1°   | ~10-100'' radius (statistical, 90% confidence) |
> | VT          | 450 - 650 nm (Blue band) <br/>650 - 1000 nm (Red band) | 26' x 26'     | ~1'' radius (statistical, 90% confidence)      |

Reviewer: ```suggestion
| ECLAIRs     | 4 - 250 keV                                            | 2 ster        | ~8ʹ radius (statistical, 90% confidence)       |
| GRM         | 15 - 5000 keV                                          | 2.6 ster      | ~1-10° radius (statistical, 90% confidence)    |
| MXT         | 0.2 - 10 keV                                           | 1.1° x 1.1°   | ~10-100ʺ radius (statistical, 90% confidence) |
| VT          | 450 - 650 nm (Blue band) <br/>650 - 1000 nm (Red band) | 26ʹ x 26'     | ~1ʺ radius (statistical, 90% confidence)      |
```

Reviewer: Use prime and double-prime symbols for arcseconds and arcminutes, not single quotes.

 </div>

#### JSON-Serialized GCN Notice Types in GCN Kafka

SVOM distributes alerts and localizations detections of gamma-ray transients. These notices are published on the GCN Kafka and VOEvent topics as `gcn.notices.svom` and `gcn.notices.svom.voevent`, respectively.

 <div className="overflow-table">

| Type               | Contents                                                                                  | Latency   |
| ------------------ | ----------------------------------------------------------------------------------------- | --------- |
| `trigger`          | eclairs.trigger.wakeup, eclairs.trigger.not-slewing, eclairs.trigger.slewing, grm.trigger | ~1-10 sec |
| `mxt.localisation` | mxt.localisation                                                                          | ~1 min    |

> | `refined`          | eclairs.refined, grm.refined, eclgrm.refined                                              | ~10 min   |

Reviewer: What are these "types" supposed to represent? They don't look like topic names.

Author: In recent conversation (see PR https://github.com/nasa-gcn/gcn-schema/pull/182), looks like different topics.
I have requested to reduce some topics.
